stdlib/2/os/__init__.pyi

```diff
@@ -191,16 +191,20 @@ def stat_float_times() -> bool: ...
 def statvfs(path: _PathType) -> _StatVFS: ...  # Unix only
 def symlink(source: _PathType, link_name: _PathType) -> None: ...
 def unlink(path: _PathType) -> None: ...
-def utime(path: _PathType, times: Optional[Tuple[int, int]]) -> None: ...
+def utime(path: _PathType, times: Optional[Tuple[float, float]]) -> None: ...
+
+# TODO onerror: function from OSError to void
 def walk(top: AnyStr, topdown: bool = ..., onerror: Any = ...,
          followlinks: bool = ...) -> Iterator[Tuple[AnyStr, List[AnyStr],
                                                     List[AnyStr]]]: ...
 
-def abort() -> None: ...
-def execl(file: _PathType, *args) -> None: ...
-def execle(file: _PathType, *args) -> None: ...
-def execlp(file: _PathType, *args) -> None: ...
-def execlpe(file: _PathType, *args) -> None: ...
+def abort() -> NoReturn: ...
+def execl(file: _PathType, arg0: Union[bytes, Text], *args: Union[bytes, Text]) -> None: ...
```

Comment:
I can't find it back now, but didn't we have a PR recently that removed the arg0 arguments from these (because they actually just take `*args`) and changed the return types to `NoReturn`?